lib/cli/commands/validate.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+const fs = require('fs-extra')
+const path = require('path')
+const pathContains = require('contains-path')
+const Ajv = require('ajv')
+
+const supportSchema = require('./../../../schema.json')
+
+module.exports = function (opts) {
+  return {
+    command: 'validate',
+    desc: 'Validate support information schema',
+    builder: function (yargs) {
+      return yargs
+    },
+    handler: async function (argv) {
+      let userPackageJson
+      try {
+        userPackageJson = await fs.readJSON(path.join(process.cwd(), 'package.json'))
+      } catch (error) {
+        argv.log.error(error)
+        return
+      }
+
+      const ajv = new Ajv({
+        // coerceTypes: true,
+        useDefaults: true,
+        removeAdditional: true,
+        allErrors: true,
+        nullable: true
+      })
+
+      let { support } = userPackageJson
+
+      if (support === true) {
+        support = await fs.readJSON(path.join(process.cwd(), 'package-support.json'))
+      } else if (typeof support === 'string') {
+        const p = path.join(process.cwd(), support)
+        if (!pathContains(process.cwd(), p)) {
+          throw new Error('Support string path must be contained in cwd')
+        }
```

Comment:
how does this handle if the string resolves to a symlink to a path outside the cwd?

Reply:
I changed how this file is read: now it is relative to the package.json we just read.

Right now the read of the package.json is forced to cwd (so nothing has changed), but if we search the package.json this code will adapt to that path